spec/motion-spec/nontrue_spec.rb

```diff
@@ -1,17 +1,16 @@
 # -*- encoding : utf-8 -*-
-motion_require '../spec_helper'
 
-$false_is_not_true = false.should.not.be.true
-$nil_is_not_true = nil.should.not.be.true
+false_is_not_true = false.should.not.be.true
+nil_is_not_true = nil.should.not.be.true
 
 describe 'A non-true value' do
-  it 'should pass negated tests inside specs' do
+  it 'passes negated tests inside specs' do
     false.should.not.be.true
     nil.should.not.be.true
   end
 
-  it 'should pass negated tests outside specs' do
-    $false_is_not_true.should.be.true
-    $nil_is_not_true.should.be.true
+  it 'passes negated tests outside specs' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.